.github/actions/setup-woocommerce-monorepo/action.yml

```diff
@@ -41,6 +41,7 @@ runs:
       with:
         php-version: ${{ inputs.php-version }}
         coverage: none
+        tools: cs2pr, phpcs
```

Comment:
Does it make sense to add a new input to this action, and default it to no tools, and just setup php with these tools where needed?

Reply:
@jonathansadowski ah yeah... mmm probably ay. How would you feel about a follow up PR for that? I can create an issue for it?

I could do it here, but I am just concerned it will balloon into more testing effort etc because inputs means logic in workflow etc and I'd like to fix the core problem first. (also how much cost is it to include the tools? I'm not sure)

Reply:
Yeah, I think that's okay. If there's not much cost to including the tools, maybe it's fine as-is. Let's create a follow-up issue, and if we determine it's not needed, we can just close it.